iphone/cli/commands/_build.js

```diff
@@ -5884,7 +5884,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 											// dest doesn't exist, or new contents differs from existing dest file
 											if (!exists || newContents !== fs.readFileSync(to).toString()) {
 												this.logger.debug(__('Copying and minifying %s => %s', from.cyan, to.cyan));
-												exists && fs.unlinkSync(to);
+												// no need to delete if it exists, writeFile will overwrite anyways
```

Comment:
Android does the same thing on line 2974.

Reply:
Yes, but there it's needed. Because on Android we try to symlink files over, and if the destination is a symlink it must be removed first or the write will actually write to the symlink target (i.e. the user's original source)